libcloud/loadbalancer/drivers/elb.py

```diff
@@ -115,6 +124,14 @@ def get_balancer(self, balancer_id):
         data = self.connection.request(ROOT, params=params).object
         return self._to_balancers(data)[0]
 
+    def get_tags(self, balancer_id):
```

Comment:
Since in most cases we either operate with single balancer instance or list of them and they should already have tags data - no, we won't :) 

Made list_balancer_tags and populate_balancer_tags private extra (_ex_list_balancer_tags & _ex_populate_balancer_tags) and removed from base.